### PR TITLE
feat: update portfolio for beets

### PR DESCRIPTION
**Pull request by groninge01**
- move veballockprovider back to lib for now (disabled for beets)
- add beets & voting apr for mabeets
- make hardcoded `Balancer` references dynamic

### CODE REVIEW

File: packages/lib/shared/hooks/useAprTooltip.ts [hunk 1]
```diff
@@ -33,14 +34,20 @@ In addition, veBAL holders can get an extra boost of up to 2.5x.`
...
+const maBeetsRewardTooltipText = 'This is the APR you will receive when a relic is fully matured.'
+
 // Types that must be added to the total base
 const TOTAL_BASE_APR_TYPES = [
   GqlPoolAprItemType.SwapFee_24H,
   GqlPoolAprItemType.IbYield,
   GqlPoolAprItemType.Staking,
   GqlPoolAprItemType.Merkl,
-  GqlPoolAprItemType.Surplus,
+  GqlPoolAprItemType.Surplus_24H,
```
Comment (garethfuller) [inline packages/lib/shared/hooks/useAprTooltip.ts:48]: Why is this changing? Are you sure it won't break the CoW APRs?

**Review by garethfuller — COMMENTED** [review]
Same comments as the other PR about project consts.

**Review by garethfuller — APPROVED** [review]
Approved to unblock, but please address comments where relevant.

File: packages/lib/shared/hooks/useAprTooltip.ts (hunk 1 above)
Comment (groninge01) [reply packages/lib/shared/hooks/useAprTooltip.ts:48]: the prop is deprecated in the api so i updated it

image

**Pull request merged**